### DISCUSSION TITLE
How to auto-delete a Queue?

### DISCUSSION THREAD

**Discussion by ZedZipDev** [rabbitmq/rabbitmq-dotnet-client#1126] [Q&A]
I have N customers, each of them has its own Queue using fanout exchange:
Que-1
Que-2
...
Que-N

Publishers published messages to to fanout exchange, i.e. Publisher->pub msg->(to all Que-1 - Que-N)

Each customer reads its own queue 
customer i - reads -Que-i

The question is: 

How can I solve this problem: if no customer  reads this Que-* in K minutes then this Que should be deleted with messages.
I.e. publishers can publish messages, but if no customer reads it then it should be dropped.

**Comment by lukebakken** [comment]
https://www.google.com/search?q=site%3Arabbitmq.com+auto+delete+queue
